enum.c

```diff
@@ -132,21 +132,28 @@ enum_grep0(VALUE obj, VALUE pat, VALUE test)
 }
 
 /*
- *  call-seq:
- *     enum.grep(pattern)                  -> array
- *     enum.grep(pattern) { |obj| block }  -> array
+ * call-seq:
+ *   grep(pattern) -> array
+ *   grep(pattern) {|element| ... } -> array
+ *
+ * Returns an array of objects based elements of +self+ that match the given pattern.
+ *
+ * With no block given, returns an array containing each element
+ * for which <tt>pattern === element</tt> is +true+:
+ *
+ *   a = %w/foo bar car moo/
```

Comment:
While this is valid, it looks odd, almost appears more like a regexp.  Usually with `%w`, you would use `%w[]` to indicate the creation of the array, or `%w""` or `%w''` to indicate a string being split.  Using another character such as `/` for `%w` would make sense if there were `[`, `]`, `'`, and/or `"` characters inside the string, but that isn't true in this case.

This is a style issue, so I'm not going to block it, but please consider whether another delimiter may be easier to understand before merging.  Same issue in other cases in this PR.

Reply:
When I'm not being lazy, I avoid %w altogether in the doc, b/c some people may not know that idiom.  These are changed to straight-up array notation.